test/queue_list_test.dart

```diff
@@ -136,14 +136,18 @@ void main() {
     });
 
     test('grows a smaller queue', () {
-      var queue = QueueList.from([1, 2, 3]);
+      var queue = QueueList<int?>.from([1, 2, 3]);
       queue.length = 5;
       expect(queue, equals([1, 2, 3, null, null]));
     });
 
     test('throws a RangeError if length is less than 0', () {
       expect(() => QueueList().length = -1, throwsRangeError);
     });
+
+    test('throws an UnsupportedError if element type is non-nullable', () {
```

Comment:
We can _only_ run this test in strong mode. Do we care? Is there a reason we might want to run tests in weak mode too for some time?

Reply:
We can cross that bridge when we come to it imo

Reply:
That bridge needs to be crossed before landing, or Travis will probably yell at you.